src/marshaller/HipieDDL.js

```diff
@@ -475,6 +475,7 @@
                 this.mappings = new TableMappings(this.visualization, source.mappings);
                 break;
             case "GRAPH":
+            case "GRAPHC":
```

Comment:
Not needed (GRAPH is a HIPIE keyword, user will set the _chartType("graph_GraohC" for this graph)